SpatialGDK/Source/SpatialView/SpatialView.Build.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+using UnrealBuildTool;
+
+public class SpatialView : ModuleRules
+{
+	public SpatialView(ReadOnlyTargetRules Target) : base(Target)
+	{
+		PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
+		bFasterWithoutUnity = true;
```

Comment:
Why do we turn unity builds off?

Reply:
No serious reason, it's here since every other module is set up this way.